brayns/io/CMakeLists.txt

```diff
@@ -34,7 +33,7 @@ if(ASSIMP_FOUND)
 endif()
 
 if(TARGET Brion)
-  list(APPEND BRAYNSIO_LINK_LIBRARIES Brion Brain)
+  list(APPEND BRAYNSIO_LINK_LIBRARIES Brion Brain ${HDF5_LIBRARIES})
 endif()
```

Comment:
All link libraries are now declared public, which means that users of BraynsIO have full access to all symbols to these libraries. One only needs to declare link libs public if they are part of the API, e.g., BraynsIO uses some boost type in its API.